addons/jam/CfgWeapons.hpp

```diff
@@ -31,10 +31,18 @@ class CfgWeapons {
         };
     };
 
+    class arifle_Katiba_Base_F : Rifle_Base_F {
+        magazineWell[] = {"CBA_65x39_Katiba"};
+    };
+
+    class arifle_ARX_base_F : Rifle_Base_F {
+        magazineWell[] = {"CBA_65x39_Katiba"};
+    };
+
     class EBR_base_F: Rifle_Long_Base_F {
         magazineWell[] = {"CBA_762x51_M14"};
     };
-    class DMR_06_base_F: Rifle_Long_Base_F {
+    class DMR_03_base_F: Rifle_Long_Base_F {
```

Comment:
This must be a mistake here. You replaced DMR_06_base_F with DMR_03_base_F

Reply:
https://github.com/CBATeam/CBA_A3/pull/1020/commits/93af9ef5fac23061f933f6ca15bd80b6fb2cbb12 This is the mistake. Commit message says it reverts the SIG 556. But it reverted the M14.

Reply:
Yep, has been fixed, thanks for spotting it!